src/screens/main-screen/main-menu-screen.ts

```
@@ -182,7 +182,7 @@ export class MainMenuScreen extends BaseGameScreen {
         return this.transitionToSettingsScreen();
 
       default:
-        return this.closeableMessageObject?.show("Invalid menu option");
+        return this.closeableMessageEntity?.show("Invalid menu option");
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)